docs/api.rst

```diff
@@ -217,6 +217,47 @@ Helper Functions
     Used for record links to add the appropriate processing annotation to the
     link.
 
+..  class::
+    ConstArray(iterator)
+
+    Used for **Constant Link Values** available since EPICS 3.16.1.
+    Constant Link Values is an EPICS feature which allows passing
+    an list of strings or a list of numbers as a constant into
+    a field which contains a DB link (e.g. INP).
+    See EPICS Release Notes (Section "Constant Link Values")
+    for more details and supported use.
+
+    ConstArray will accept any iterable (e.g. a list) which can generate
+    a non-empty list of values of the same type. Allowed types are:
+
+    * strings and parameters (i.e. :py:class:`epicsdbbuilder.Parameter`)
+
+    * numbers (integers, floating-point, :py:class:`Decimal.Decimal`, and booleans.
+      Booleans will convert to 0 (false) or 1 (true) automatically.
+
+
+    Known limitations:
+
+    * No field type or record type check.
+      ConstArray can be assigned to any field despite the field
+      or the record type does not support Constant Link Values.
+      Use it with link fiels (e.g. INP) of record types stringin,
+      stringout, lso, lsi, printf, waveform, subArray, and aai.
+      Any other use is undefined and a warning may or may not appear
+      while loading the DB
+      (e.g. assigning ["1.23"] to INP of the record type ai will print a warning
```

Comment:
Put `` ` `` markup around constant strings `["1.23"]` and `[1.23]`.

Also, 80 column line wrapping, please!  (Yes, I _did_ program on punched cards at university.)

Reply:
I am aware I keep running into it. I am sorry. I understand the reason for 80 columns especially when reviewing code with side-by-side view. I have set a vertical column mark to help me to see it clearly.

Reply:
NP 😛